src/components/LogsView/LogsView.css

```diff
@@ -6,9 +6,7 @@
   .logContainer {
     margin: 10px;
     text-align: end;
-  }
-  @media (max-width: 1199px) {
-    .logContainer {
```

Comment:
This needs to be reintroduced. To explicitly specify the selector in media-query rule.

After removing sass, we can no more nest properties; unless we introduce support to nesting. It threw many build warnings and that is why I had removed them. They were just warnings, and not errors, but still.

Reply:
Ok! I'll fix it

Reply:
Done